loadtest/control/simulcontroller/utils.go

```diff
@@ -49,7 +50,17 @@ func genMessage(isReply bool) string {
 	// TODO: make a util function out of this behaviour.
 	wordCount := rand.Intn(avgWordCount*2-minWordCount*2) + minWordCount
 
-	return control.GenerateRandomSentences(wordCount)
+	message := control.GenerateRandomSentences(wordCount)
+
+	// 2% of the times someone is mentioned.
+	if rand.Float64() < 0.02 {
```

Comment:
This is going to be higher but no need to change for now. I can update on one of my refining PRs when I get some actual data.